HarmonyCore/FileIO/Queryable/PreparedQueryPlan.dbl

```diff
@@ -861,6 +861,40 @@ namespace Harmony.Core.FileIO.Queryable
 			mreturn orderByResult
 		endmethod
 
+        ;; A helper method that verifies version of runtime and SRV_COMPAT 
+        private method VerifyVersionCompatibility, boolean
+            isRemote, Nullable<Boolean>
+        proc
+            data condition, boolean, false
+            data runtimeVer = SYN_VERSION
+            data splitVer = runtimeVer.Split('.')
+            data versnNum = Int32.Parse(splitVer[1])
+            data majorRevNum = Int32.Parse(splitVer[2])
+            data versionCheck, boolean, false
+            if(versnNum > 12 || (versnNum = 12 && majorRevNum >= 2)) then
```

Comment:
The check for version and revision number could potentially be replaced with a check for a define DBLV123(12.2 features) when it becomes available.

Reply:
That define will only determine that we build Harmony Core with a 12.2 compiler, it wont tell us anything about the customer projects where we're being consumed